api/src/main/java/cz/muni/fi/pa165/dto/StoreDTO.java

```diff
@@ -64,10 +64,8 @@ public boolean equals(Object other) {
 
     @Override
     public int hashCode() {
-        int result = getId().hashCode();
-        result = 31 * result + (getName() != null ? getName().hashCode() : 0);
-        return result;
-    }
+        return name.hashCode() * 21;
```

Comment:
Myslim ze nie

http://stackoverflow.com/questions/6981968/java-string-hashcode-of-null-string